Airports/EGLL/Basic.txt

```diff
@@ -1,3 +1,3 @@
 London Heathrow
 N051.28.39.000 W000.27.41.000
-119.725
+119.730
```

Comment:
believe this should be the AIR S frequency @PLM1995 @hazzas-99 ?

Reply:
Yeah I don't know what this is used for, if anything, suspect it's one of those we don't care about too much, but I think it should officially be the AIR freq. From our WIKI:
"Airport Frequency - according to the VRC wiki, this is supposed to be the TWR frequency, but it varies across our sector file, and doesn't (appear to) make any difference to controlling..."